icons/tortoisegit/tortoisegit-plain.svg

```diff
@@ -0,0 +1 @@
2a2.14 2.14 0 0 0-.682.011c-.452.08-.891.28-1.359.578l-.068.043-.058.057c-1.408 1.359-2.255 2.224-2.844 3.825-.584 1.588-.948 3.828-1.679 8.17-.785 2.174-1.929 4.066-3.549 5.329-1.633 1.273-3.752 1.96-6.701 1.651l-.298-.031-.246.171c-2.906 2.024-4.062 4.77-4.737 7.686s-.939 6.046-2.059 9.21l-.271.766.777.253c5.048 1.647 8.377 1.53 10.763.672s3.76-2.409 4.887-3.286l.027-.022.026-.024c1.233-1.146 1.977-2.492 2.978-3.526s2.2-1.827 4.547-2.03c3.871-.06 6.416-.23 8.234-.791 1.651-.51 2.685-1.493 3.269-2.808 1.938-.59 3.218-.383 4.68-.671.77-.152 1.569-.489 2.342-1.158.767-.664 1.525-1.636 2.392-3.086.556-.83.685-1.947.152-3.005s-1.572-2.17-3.292-3.693l.079.081c-.956-1.123-2.49-1.322-3.783-1.536-.647-.107-1.249-.199-1.676-.314s-.599-.282-.567-.227l-.421-.703-.696.438c-1.636 1.031-2.909 1.534-3.835 1.673s-1.478-.035-1.938-.398c-.92-.724-1.464-2.612-1.727-4.782s-.296-4.513-.734-6.229c-.219-.858-.529-1.652-1.301-2.083a1.85 1.85 0 0 0-.633-.211zm-.401 1.578c.159-.028.197-.009.249.02.105.059.359.4.534 1.086.35 1.373.419 3.75.696 6.029s.708 4.567 2.323 5.838c.807.636 1.912.914 3.172.724 1.121-.169 2.422-.732 3.92-1.617a3.46 3.46 0 0 0 .941.399c.571.153 1.202.243 1.831.347 1.258.208 2.443.547 2.823.994l.037.043.042.037c1.663 1.473 2.578 2.528 2.925 3.217s.289.905-.054 1.414l-.012.018-.011.019c-.825 1.384-1.507 2.226-2.087 2.727s-1.041.687-1.602.798c-1.123.222-2.758-.019-5.185.804l-.37.125-.128.368c-.418 1.201-.933 1.761-2.429 2.223s-3.97.663-7.835.722h-.029l-.028.002c-2.688.231-4.381 1.287-5.568 2.513s-1.927 2.545-2.916 3.465c-1.297 1.02-2.432 2.294-4.393 2.999-1.873.674-4.619.813-8.969-.488.971-3.091 1.279-6.032 1.865-8.561.621-2.681 1.549-4.856 3.926-6.591 3.103.243 5.607-.527 7.483-1.989 1.954-1.523 3.239-3.716 4.09-6.098l.046-.129v-.07c.734-4.359 1.12-6.58 1.607-7.907.479-1.304 1.051-1.869 2.381-3.153.317-.193.576-.302.725-.328z"/><path d="M125.145 37.348c0-4.719-2.422-8.555-5.406-8.555s-5.402 3.836-5.402 8.555c0 4.727 2.418 8.551 5.402 8.551s5.406-3.824 5.406-8.551zm0 0" fill="#0078ba"/><path d="M119.739 27.998c-1.837 0-3.424 1.178-4.499 2.881s-1.705 3.975-1.705 6.47.63 4.769 1.705 6.471 2.662 2.876 4.499 2.876 3.424-1.174 4.5-2.875 1.707-3.973 1.707-6.471-.631-4.767-1.707-6.47-2.662-2.88-4.5-2.88zm0 1.591c1.147 0 2.263.74 3.144 2.135s1.461 3.4 1.461 5.624-.58 4.232-1.461 5.624-1.996 2.131-3.144 2.131-2.262-.738-3.142-2.13-1.46-3.396-1.46-5.625.58-4.23 1.46-5.624 1.995-2.135 3.142-2.135z"/><path d="M118.84 38.441c0-4.719-2.418-8.547-5.406-8.547s-5.402 3.828-5.402 8.547 2.418 8.551 5.402 8.551 5.406-3.828 5.406-8.551zm0 0" fill="#0078ba"/><path d="M113.434 29.099c-1.837 0-3.423 1.175-4.499 2.876s-1.705 3.972-1.705 6.466.63 4.768 1.705 6.47 2.662 2.877 4.499 2.877 3.426-1.175 4.502-2.877 1.705-3.973 1.705-6.47-.63-4.765-1.706-6.466-2.663-2.876-4.501-2.876zm0 1.591c1.15 0 2.265.739 3.145 2.132s1.46 3.396 1.46 5.62-.58 4.23-1.46 5.623-1.995 2.132-3.145 2.132-2.262-.739-3.142-2.132-1.46-3.397-1.46-5.623.58-4.227 1.46-5.62 1.995-2.132 3.142-2.132z"/></g></svg>
```

Comment:
For the `plain` version, please only have one color in the whole SVG. I think the the blue of the shell is a good choice. Please color/fill the entire turtle with it. You can compare [`docker-original`](https://github.com/devicons/devicon/blob/master/icons/docker/docker-original.svg) and [`docker-plain`](https://github.com/devicons/devicon/blob/master/icons/docker/docker-plain.svg) to see what we want to achieve. Don't worry about the transparent belly of `docker`, just the same color throughout is good enough.

Also, please make the outlines/strokes of the turtle transparent as well. You can check out [this](https://github.com/devicons/devicon/blob/master/icons/android/android-plain.svg) and see the gap between the arms and the body. It'll be wonderful if you can do it like that.